website/docs/r/network_manager_admin_rule.html.markdown

```diff
@@ -0,0 +1,136 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_manager_admin_rule"
+description: |-
+  Manages a Network Manager Admin Rule.
+---
+
+# azurerm_network_manager_admin_rule
+
+Manages a Network Manager Admin Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+data "azurerm_subscription" "current" {
+}
+
+resource "azurerm_network_manager" "example" {
+  name                = "example-network-manager"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  scope {
+    subscription_ids = [data.azurerm_subscription.current.id]
+  }
+  scope_accesses = ["Connectivity", "SecurityAdmin"]
+  description    = "example network manager"
+}
+
+resource "azurerm_network_manager_network_group" "example" {
+  name               = "example-network-group"
+  network_manager_id = azurerm_network_manager.example.id
+}
+
+resource "azurerm_network_manager_security_admin_configuration" "example" {
+  name               = "example-admin-conf"
+  network_manager_id = azurerm_network_manager.example.id
+}
+
+resource "azurerm_network_manager_admin_rule_collection" "example" {
+  name                            = "example-admin-rule-collection"
+  security_admin_configuration_id = azurerm_network_manager_security_admin_configuration.example.id
+  network_group_ids               = [azurerm_network_manager_network_group.example.id]
+}
+
+resource "azurerm_network_manager_admin_rule" "example" {
+  name                     = "example-admin-rule"
+  admin_rule_collection_id = azurerm_network_manager_admin_rule_collection.example.id
+  access                   = "Deny"
+  direction                = "Outbound"
+  priority                 = 1
+  protocol                 = "Tcp"
+  source_port_ranges       = ["80"]
+  destination_port_ranges  = ["80"]
+  source {
+    address_prefix_type = "ServiceTag"
+    address_prefix      = "Internet"
+  }
+  destination {
+    address_prefix_type = "IPPrefix"
+    address_prefix      = "*"
+  }
+  description = "example admin rule"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Network Manager Admin Rule. Changing this forces a new Network Manager Admin Rule to be created.
+
+* `admin_rule_collection_id` - (Required) Specifies the ID of the Network Manager Admin Rule Collection. Changing this forces a new Network Manager Admin Rule to be created.
+
+* `access` - (Required) Specifies the access allowed for this Network Manager Admin Rule. Possible values are `Allow`, `AlwaysAllow`, and `Deny`.
```

Comment:
should this be action not access to match other resources?

Reply:
changed to `action`